autohdr.sh

```diff
@@ -16,12 +16,17 @@ echo "Processing directory $DIR"
 OUTPUT_FILE=$(ls $DIR/*.JPG | head -n 1 | sed -e 's/\.JPG$/_hdr.pto/')
 echo "creating '$OUTPUT_FILE'..."
 
-pto_gen $DIR/*.JPG -o $OUTPUT_FILE
+NUM_IMAGES=$(ls $DIR/*.JPG -l | wc -l)
+
+
+pto_gen $DIR/*.JPG -s $NUM_IMAGES -o $OUTPUT_FILE
```

Comment:
That is fine with me, we should probably add the option on auto_pano_and_hdr.sh too